app/src/main/java/com/prangesoftwaresolutions/audioanchor/PlayActivity.java

```diff
@@ -813,6 +799,25 @@ public void onStopTrackingTouch(SeekBar seekBar) {
             }
         });
 
+        final ImageView resetIV = dialogView.findViewById(R.id.reset);
+        resetIV.setOnClickListener(v -> {
+            int speed = Integer.parseInt(getString(R.string.preference_playback_speed_default));
+            float speedFloat = (float) (speed / 10.0);
+            playbackSpeedTV.setText(getResources().getString(R.string.playback_speed_label, speedFloat));
+
+            // Store new playback speed in shared preferences
+            SharedPreferences.Editor editor = mSharedPreferences.edit();
+            editor.putInt(getString(R.string.preference_playback_speed_key), speed);
+            editor.apply();
+
+            // Set position of seeker to default position 1.0
+            playbackSpeedSB.setProgress(getProgressFromPlaybackSpeed((int) (1.0 * 10.0)));
```

Comment:
You can just use the variable `speed` which is already an `int` and has the value we need here.

Reply:
Exactly! I missed a possibility of use variable `speed`. I have already fixed it.